qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/QuteValidationSettings.java

```diff
@@ -30,13 +38,15 @@
 
 	private boolean enabled;
 
+	private QuteValidationTypeSettings undefinedVariable;

```

Comment:
Replace equals and hashCode with:

	```
@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + (enabled ? 1231 : 1237);
		result = prime * result + ((undefinedVariable == null) ? 0 : undefinedVariable.hashCode());
		return result;
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (obj == null)
			return false;
		if (getClass() != obj.getClass())
			return false;
		QuteValidationSettings other = (QuteValidationSettings) obj;
		if (enabled != other.enabled)
			return false;
		if (undefinedVariable == null) {
			if (other.undefinedVariable != null)
				return false;
		} else if (!undefinedVariable.equals(other.undefinedVariable))
			return false;
		return true;
	}
```

Please note that this code has been generated by Eclipse IDE "Source / Generate hashCode() and equals()..."